index.html

```diff
@@ -0,0 +1,291 @@
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="container my-5 py-5" id="contact">
+      <div class="row justify-content-between">
+        <h2 class="display-6 my-auto fw-bold text-center col-sm-12 col-md-6 col-lg-6 co-xl-6"> I'm always interested in
+          hearing<br>about new projects, so if you'd<br>like to chat please get in touch.</h2>
+        <form action="#" method="post" class="col-sm-12 col-md-6 col-lg-6 co-xl-6">
+          <div class="row my-3 d-sm-none d-xs-none d-md-flex d-lg-flex d-xl-flex">
+            <div class="col-6"><input type="text" placeholder="First Name" required class="form-control"></div>
+            <div class="col-6"><input type="text" placeholder="Last Name" required class="form-control"></div>
+          </div>
+          <input type="text" placeholder="Full Name" required class="form-control d-md-none d-lg-none d-xl-none my-3">
+          <input type="email" placeholder="Email address" required class="form-control my-3">
+          <textarea cols="30" rows="10" placeholder="Typing somthing" class="form-control"></textarea>
+          <button type="submit" class="btn btn-success my-4">Get in touch</button>
+        </form>
+      </div>
+    </section>
+    <div class="modal fade" id="projectmodal" tabindex="-1" aria-hidden="true">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+          <div class="card">
+            <img class="card-img-top" src="./assets/img/project.png" alt="">
+            <div class="card-body border-top text-center my-3">
+              <h3>Multi-Post Stories Gain+Glory</h3>
+              <ul class="row justify-content-left">
+                <li class="col m-2 badge bg-primary">Html</li>
+                <li class="col m-2 badge bg-primary">Css</li>
+                <li class="col m-2 badge bg-primary">Javascript</li>
+              </ul>
+              <p class="text-start"> Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea </p>
+              <div class="d-flex justify-content-between">
+                <button type="button" class="btn btn-success">
+                  see live <i class="fa fa-external-link"></i>
+                </button>
+                <button type="button" class="btn btn-success">
+                  see sources <i class="fa fa-github"></i>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
```

Comment:
[OPTIONAL] Kindly remove the unnecessary blank lines, this would make your code look cleaner.